src/template-tags/tickets.php

```diff
@@ -109,6 +109,164 @@ function tribe_events_count_available_tickets( $event = null ) {
 	}
 }//end if
 
+if ( ! function_exists( 'tribe_events_count_tickets_and_rsvp' ) ) {
+
+	/**
+	 * Returns Ticket and RSVP Count for an Event
+	 *
+	 * @param $event_id
+	 *
+	 * @return array
+	 */
+	function tribe_events_count_tickets_and_rsvp( $event_id ) {
```

Comment:
The function prefix convention for ticket functions is `tribe_tickets_` despite there being a lot of functions inherited from the old event plugins with `tribe_events_` as the prefix. New functions should use the desired prefix.